src/scenes/gameplay-scene.ts

```diff
@@ -320,7 +322,9 @@ export class GameplayScene {
     if (stoneLetter) {
       this.pickedStoneObject = stoneLetter;
       this.pickedStone = stoneLetter;
-      this.audioPlayer.playAudio(AUDIO_PATH_ON_DRAG); // Note: When refactoring, this should be moved alongside the StoneHandler and handled within that class.
+      if (this.pickedStone.frame > 99) {
+        this.audioPlayer.playAudio(AUDIO_PATH_ON_DRAG); // Note: When refactoring, this should be moved alongside the StoneHandler and handled within that class.
+      }
```

Comment:
Same comment from above.

Reply:
@bernhardccodev done. i moved it already. thank you